Description: Upstream changes introduced in version 2.2.4-2
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 slang2 (2.2.4-2) unstable; urgency=low
 .
   * Call make static to build library directly in rules, rather than
     accidentally rely on it being called by 'make test'. Closes: #623933.
 .
 The person named in the Author field signed this changelog entry.
Author: Alastair McKinstry <mckinstry@debian.org>
Bug-Debian: http://bugs.debian.org/623933

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ slang2-2.2.4/config.log
@@ -0,0 +1,1594 @@
+This file contains any messages produced by compilers while
+running configure, to aid debugging if configure makes a mistake.
+
+It was created by configure, which was
+generated by GNU Autoconf 2.67.  Invocation command line was
+
+  $ ./configure --build=i486-linux-gnu --prefix=/usr --includedir=${prefix}/include --mandir=${prefix}/share/man --infodir=${prefix}/share/info --sysconfdir=/etc --localstatedir=/var --libexecdir=${prefix}/lib/slang2 --disable-maintainer-mode --disable-dependency-tracking
+
+## --------- ##
+## Platform. ##
+## --------- ##
+
+hostname = debian
+uname -m = i686
+uname -r = 2.6.38-2-686
+uname -s = Linux
+uname -v = #1 SMP Thu Apr 7 05:24:21 UTC 2011
+
+/usr/bin/uname -p = unknown
+/bin/uname -X     = unknown
+
+/bin/arch              = unknown
+/usr/bin/arch -k       = unknown
+/usr/convex/getsysinfo = unknown
+/usr/bin/hostinfo      = unknown
+/bin/machine           = unknown
+/usr/bin/oslevel       = unknown
+/bin/universe          = unknown
+
+PATH: /usr/sbin
+PATH: /usr/bin
+PATH: /sbin
+PATH: /bin
+PATH: /usr/bin/X11
+
+
+## ----------- ##
+## Core tests. ##
+## ----------- ##
+
+configure:2468: checking build system type
+configure:2482: result: i486-pc-linux-gnu
+configure:2502: checking host system type
+configure:2515: result: i486-pc-linux-gnu
+configure:2601: checking for gcc
+configure:2617: found /usr/bin/gcc
+configure:2628: result: gcc
+configure:2857: checking for C compiler version
+configure:2866: gcc --version >&5
+gcc (Debian 4.5.2-11) 4.5.2
+Copyright (C) 2010 Free Software Foundation, Inc.
+This is free software; see the source for copying conditions.  There is NO
+warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+
+configure:2877: $? = 0
+configure:2866: gcc -v >&5
+Using built-in specs.
+COLLECT_GCC=gcc
+COLLECT_LTO_WRAPPER=/usr/lib/gcc/i486-linux-gnu/4.5.2/lto-wrapper
+Target: i486-linux-gnu
+Configured with: ../src/configure -v --with-pkgversion='Debian 4.5.2-11' --with-bugurl=file:///usr/share/doc/gcc-4.5/README.Bugs --enable-languages=c,c++,fortran,objc,obj-c++ --prefix=/usr --program-suffix=-4.5 --enable-shared --enable-multiarch --with-multiarch-defaults=i386-linux-gnu --enable-linker-build-id --with-system-zlib --libexecdir=/usr/lib --without-included-gettext --enable-threads=posix --with-gxx-include-dir=/usr/include/c++/4.5 --libdir=/usr/lib --enable-nls --enable-clocale=gnu --enable-libstdcxx-debug --enable-libstdcxx-time=yes --enable-plugin --enable-gold --enable-ld=default --with-plugin-ld=ld.gold --enable-objc-gc --enable-targets=all --with-arch-32=i586 --with-tune=generic --enable-checking=release --build=i486-linux-gnu --host=i486-linux-gnu --target=i486-linux-gnu
+Thread model: posix
+gcc version 4.5.2 (Debian 4.5.2-11) 
+configure:2877: $? = 0
+configure:2866: gcc -V >&5
+gcc: '-V' option must have argument
+configure:2877: $? = 1
+configure:2866: gcc -qversion >&5
+gcc: unrecognized option '-qversion'
+gcc: no input files
+configure:2877: $? = 1
+configure:2897: checking whether the C compiler works
+configure:2919: gcc -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c  >&5
+configure:2923: $? = 0
+configure:2971: result: yes
+configure:2974: checking for C compiler default output file name
+configure:2976: result: a.out
+configure:2982: checking for suffix of executables
+configure:2989: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c  >&5
+configure:2993: $? = 0
+configure:3015: result: 
+configure:3037: checking whether we are cross compiling
+configure:3045: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c  >&5
+configure:3049: $? = 0
+configure:3056: ./conftest
+configure:3060: $? = 0
+configure:3075: result: no
+configure:3080: checking for suffix of object files
+configure:3102: gcc -c -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2  conftest.c >&5
+configure:3106: $? = 0
+configure:3127: result: o
+configure:3131: checking whether we are using the GNU C compiler
+configure:3150: gcc -c -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2  conftest.c >&5
+configure:3150: $? = 0
+configure:3159: result: yes
+configure:3168: checking whether gcc accepts -g
+configure:3188: gcc -c -g  conftest.c >&5
+configure:3188: $? = 0
+configure:3229: result: yes
+configure:3246: checking for gcc option to accept ISO C89
+configure:3310: gcc  -c -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2  conftest.c >&5
+configure:3310: $? = 0
+configure:3323: result: none needed
+configure:3349: checking how to run the C preprocessor
+configure:3380: gcc -E  conftest.c
+configure:3380: $? = 0
+configure:3394: gcc -E  conftest.c
+conftest.c:9:28: fatal error: ac_nonexistent.h: No such file or directory
+compilation terminated.
+configure:3394: $? = 1
+configure: failed program was:
+| /* confdefs.h */
+| #define PACKAGE_NAME ""
+| #define PACKAGE_TARNAME ""
+| #define PACKAGE_VERSION ""
+| #define PACKAGE_STRING ""
+| #define PACKAGE_BUGREPORT ""
+| #define PACKAGE_URL ""
+| /* end confdefs.h.  */
+| #include <ac_nonexistent.h>
+configure:3419: result: gcc -E
+configure:3439: gcc -E  conftest.c
+configure:3439: $? = 0
+configure:3453: gcc -E  conftest.c
+conftest.c:9:28: fatal error: ac_nonexistent.h: No such file or directory
+compilation terminated.
+configure:3453: $? = 1
+configure: failed program was:
+| /* confdefs.h */
+| #define PACKAGE_NAME ""
+| #define PACKAGE_TARNAME ""
+| #define PACKAGE_VERSION ""
+| #define PACKAGE_STRING ""
+| #define PACKAGE_BUGREPORT ""
+| #define PACKAGE_URL ""
+| /* end confdefs.h.  */
+| #include <ac_nonexistent.h>
+configure:3482: checking for grep that handles long lines and -e
+configure:3540: result: /bin/grep
+configure:3545: checking for egrep
+configure:3607: result: /bin/grep -E
+configure:3612: checking for ANSI C header files
+configure:3632: gcc -c -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2  conftest.c >&5
+configure:3632: $? = 0
+configure:3705: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c  >&5
+configure:3705: $? = 0
+configure:3705: ./conftest
+configure:3705: $? = 0
+configure:3716: result: yes
+configure:3729: checking for sys/types.h
+configure:3729: gcc -c -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2  conftest.c >&5
+configure:3729: $? = 0
+configure:3729: result: yes
+configure:3729: checking for sys/stat.h
+configure:3729: gcc -c -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2  conftest.c >&5
+configure:3729: $? = 0
+configure:3729: result: yes
+configure:3729: checking for stdlib.h
+configure:3729: gcc -c -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2  conftest.c >&5
+configure:3729: $? = 0
+configure:3729: result: yes
+configure:3729: checking for string.h
+configure:3729: gcc -c -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2  conftest.c >&5
+configure:3729: $? = 0
+configure:3729: result: yes
+configure:3729: checking for memory.h
+configure:3729: gcc -c -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2  conftest.c >&5
+configure:3729: $? = 0
+configure:3729: result: yes
+configure:3729: checking for strings.h
+configure:3729: gcc -c -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2  conftest.c >&5
+configure:3729: $? = 0
+configure:3729: result: yes
+configure:3729: checking for inttypes.h
+configure:3729: gcc -c -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2  conftest.c >&5
+configure:3729: $? = 0
+configure:3729: result: yes
+configure:3729: checking for stdint.h
+configure:3729: gcc -c -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2  conftest.c >&5
+configure:3729: $? = 0
+configure:3729: result: yes
+configure:3729: checking for unistd.h
+configure:3729: gcc -c -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2  conftest.c >&5
+configure:3729: $? = 0
+configure:3729: result: yes
+configure:3742: checking minix/config.h usability
+configure:3742: gcc -c -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2  conftest.c >&5
+conftest.c:52:26: fatal error: minix/config.h: No such file or directory
+compilation terminated.
+configure:3742: $? = 1
+configure: failed program was:
+| /* confdefs.h */
+| #define PACKAGE_NAME ""
+| #define PACKAGE_TARNAME ""
+| #define PACKAGE_VERSION ""
+| #define PACKAGE_STRING ""
+| #define PACKAGE_BUGREPORT ""
+| #define PACKAGE_URL ""
+| #define STDC_HEADERS 1
+| #define HAVE_SYS_TYPES_H 1
+| #define HAVE_SYS_STAT_H 1
+| #define HAVE_STDLIB_H 1
+| #define HAVE_STRING_H 1
+| #define HAVE_MEMORY_H 1
+| #define HAVE_STRINGS_H 1
+| #define HAVE_INTTYPES_H 1
+| #define HAVE_STDINT_H 1
+| #define HAVE_UNISTD_H 1
+| /* end confdefs.h.  */
+| #include <stdio.h>
+| #ifdef HAVE_SYS_TYPES_H
+| # include <sys/types.h>
+| #endif
+| #ifdef HAVE_SYS_STAT_H
+| # include <sys/stat.h>
+| #endif
+| #ifdef STDC_HEADERS
+| # include <stdlib.h>
+| # include <stddef.h>
+| #else
+| # ifdef HAVE_STDLIB_H
+| #  include <stdlib.h>
+| # endif
+| #endif
+| #ifdef HAVE_STRING_H
+| # if !defined STDC_HEADERS && defined HAVE_MEMORY_H
+| #  include <memory.h>
+| # endif
+| # include <string.h>
+| #endif
+| #ifdef HAVE_STRINGS_H
+| # include <strings.h>
+| #endif
+| #ifdef HAVE_INTTYPES_H
+| # include <inttypes.h>
+| #endif
+| #ifdef HAVE_STDINT_H
+| # include <stdint.h>
+| #endif
+| #ifdef HAVE_UNISTD_H
+| # include <unistd.h>
+| #endif
+| #include <minix/config.h>
+configure:3742: result: no
+configure:3742: checking minix/config.h presence
+configure:3742: gcc -E  conftest.c
+conftest.c:19:26: fatal error: minix/config.h: No such file or directory
+compilation terminated.
+configure:3742: $? = 1
+configure: failed program was:
+| /* confdefs.h */
+| #define PACKAGE_NAME ""
+| #define PACKAGE_TARNAME ""
+| #define PACKAGE_VERSION ""
+| #define PACKAGE_STRING ""
+| #define PACKAGE_BUGREPORT ""
+| #define PACKAGE_URL ""
+| #define STDC_HEADERS 1
+| #define HAVE_SYS_TYPES_H 1
+| #define HAVE_SYS_STAT_H 1
+| #define HAVE_STDLIB_H 1
+| #define HAVE_STRING_H 1
+| #define HAVE_MEMORY_H 1
+| #define HAVE_STRINGS_H 1
+| #define HAVE_INTTYPES_H 1
+| #define HAVE_STDINT_H 1
+| #define HAVE_UNISTD_H 1
+| /* end confdefs.h.  */
+| #include <minix/config.h>
+configure:3742: result: no
+configure:3742: checking for minix/config.h
+configure:3742: result: no
+configure:3763: checking whether it is safe to define __EXTENSIONS__
+configure:3781: gcc -c -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2  conftest.c >&5
+configure:3781: $? = 0
+configure:3788: result: yes
+configure:3803: checking whether gcc needs -traditional
+configure:3837: result: no
+configure:3849: checking for library containing strerror
+configure:3880: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c  >&5
+configure:3880: $? = 0
+configure:3897: result: none required
+configure:3923: checking C compiler that understands ANSI prototypes
+configure:3937: gcc -c -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2  conftest.c >&5
+configure:3937: $? = 0
+configure:3939: result: gcc looks ok.  Good.
+configure:3950: checking whether make sets $(MAKE)
+configure:3972: result: yes
+configure:4024: checking for ranlib
+configure:4040: found /usr/bin/ranlib
+configure:4051: result: ranlib
+configure:4087: checking for a BSD-compatible install
+configure:4155: result: /usr/bin/install -c
+configure:4185: checking terminfo layout
+configure:4193: result: default
+configure:4209: checking type of readline support for slsh
+configure:4221: result: slang
+configure:4229: checking for X
+configure:4337: gcc -E  conftest.c
+configure:4337: $? = 0
+configure:4368: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c -lX11  >&5
+configure:4368: $? = 0
+configure:4418: result: libraries , headers 
+configure:4517: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c   -lX11 >&5
+configure:4517: $? = 0
+configure:4615: checking for gethostbyname
+configure:4615: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c  >&5
+configure:4615: $? = 0
+configure:4615: result: yes
+configure:4712: checking for connect
+configure:4712: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c  >&5
+configure:4712: $? = 0
+configure:4712: result: yes
+configure:4761: checking for remove
+configure:4761: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c  >&5
+configure:4761: $? = 0
+configure:4761: result: yes
+configure:4810: checking for shmat
+configure:4810: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c  >&5
+configure:4810: $? = 0
+configure:4810: result: yes
+configure:4868: checking for IceConnectionNumber in -lICE
+configure:4893: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c -lICE   >&5
+configure:4893: $? = 0
+configure:4902: result: yes
+configure:4943: checking for stdlib.h
+configure:4943: result: yes
+configure:4943: checking for unistd.h
+configure:4943: result: yes
+configure:4943: checking for memory.h
+configure:4943: result: yes
+configure:4943: checking termios.h usability
+configure:4943: gcc -c -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2  conftest.c >&5
+configure:4943: $? = 0
+configure:4943: result: yes
+configure:4943: checking termios.h presence
+configure:4943: gcc -E  conftest.c
+configure:4943: $? = 0
+configure:4943: result: yes
+configure:4943: checking for termios.h
+configure:4943: result: yes
+configure:4943: checking malloc.h usability
+configure:4943: gcc -c -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2  conftest.c >&5
+configure:4943: $? = 0
+configure:4943: result: yes
+configure:4943: checking malloc.h presence
+configure:4943: gcc -E  conftest.c
+configure:4943: $? = 0
+configure:4943: result: yes
+configure:4943: checking for malloc.h
+configure:4943: result: yes
+configure:4943: checking locale.h usability
+configure:4943: gcc -c -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2  conftest.c >&5
+configure:4943: $? = 0
+configure:4943: result: yes
+configure:4943: checking locale.h presence
+configure:4943: gcc -E  conftest.c
+configure:4943: $? = 0
+configure:4943: result: yes
+configure:4943: checking for locale.h
+configure:4943: result: yes
+configure:4943: checking langinfo.h usability
+configure:4943: gcc -c -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2  conftest.c >&5
+configure:4943: $? = 0
+configure:4943: result: yes
+configure:4943: checking langinfo.h presence
+configure:4943: gcc -E  conftest.c
+configure:4943: $? = 0
+configure:4943: result: yes
+configure:4943: checking for langinfo.h
+configure:4943: result: yes
+configure:4943: checking fcntl.h usability
+configure:4943: gcc -c -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2  conftest.c >&5
+configure:4943: $? = 0
+configure:4943: result: yes
+configure:4943: checking fcntl.h presence
+configure:4943: gcc -E  conftest.c
+configure:4943: $? = 0
+configure:4943: result: yes
+configure:4943: checking for fcntl.h
+configure:4943: result: yes
+configure:4943: checking sys/fcntl.h usability
+configure:4943: gcc -c -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2  conftest.c >&5
+configure:4943: $? = 0
+configure:4943: result: yes
+configure:4943: checking sys/fcntl.h presence
+configure:4943: gcc -E  conftest.c
+configure:4943: $? = 0
+configure:4943: result: yes
+configure:4943: checking for sys/fcntl.h
+configure:4943: result: yes
+configure:4943: checking for sys/types.h
+configure:4943: result: yes
+configure:4943: checking sys/wait.h usability
+configure:4943: gcc -c -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2  conftest.c >&5
+configure:4943: $? = 0
+configure:4943: result: yes
+configure:4943: checking sys/wait.h presence
+configure:4943: gcc -E  conftest.c
+configure:4943: $? = 0
+configure:4943: result: yes
+configure:4943: checking for sys/wait.h
+configure:4943: result: yes
+configure:4943: checking sys/utsname.h usability
+configure:4943: gcc -c -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2  conftest.c >&5
+configure:4943: $? = 0
+configure:4943: result: yes
+configure:4943: checking sys/utsname.h presence
+configure:4943: gcc -E  conftest.c
+configure:4943: $? = 0
+configure:4943: result: yes
+configure:4943: checking for sys/utsname.h
+configure:4943: result: yes
+configure:4943: checking sys/times.h usability
+configure:4943: gcc -c -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2  conftest.c >&5
+configure:4943: $? = 0
+configure:4943: result: yes
+configure:4943: checking sys/times.h presence
+configure:4943: gcc -E  conftest.c
+configure:4943: $? = 0
+configure:4943: result: yes
+configure:4943: checking for sys/times.h
+configure:4943: result: yes
+configure:4943: checking sys/time.h usability
+configure:4943: gcc -c -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2  conftest.c >&5
+configure:4943: $? = 0
+configure:4943: result: yes
+configure:4943: checking sys/time.h presence
+configure:4943: gcc -E  conftest.c
+configure:4943: $? = 0
+configure:4943: result: yes
+configure:4943: checking for sys/time.h
+configure:4943: result: yes
+configure:4943: checking utime.h usability
+configure:4943: gcc -c -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2  conftest.c >&5
+configure:4943: $? = 0
+configure:4943: result: yes
+configure:4943: checking utime.h presence
+configure:4943: gcc -E  conftest.c
+configure:4943: $? = 0
+configure:4943: result: yes
+configure:4943: checking for utime.h
+configure:4943: result: yes
+configure:4943: checking floatingpoint.h usability
+configure:4943: gcc -c -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2  conftest.c >&5
+conftest.c:72:27: fatal error: floatingpoint.h: No such file or directory
+compilation terminated.
+configure:4943: $? = 1
+configure: failed program was:
+| /* confdefs.h */
+| #define PACKAGE_NAME ""
+| #define PACKAGE_TARNAME ""
+| #define PACKAGE_VERSION ""
+| #define PACKAGE_STRING ""
+| #define PACKAGE_BUGREPORT ""
+| #define PACKAGE_URL ""
+| #define STDC_HEADERS 1
+| #define HAVE_SYS_TYPES_H 1
+| #define HAVE_SYS_STAT_H 1
+| #define HAVE_STDLIB_H 1
+| #define HAVE_STRING_H 1
+| #define HAVE_MEMORY_H 1
+| #define HAVE_STRINGS_H 1
+| #define HAVE_INTTYPES_H 1
+| #define HAVE_STDINT_H 1
+| #define HAVE_UNISTD_H 1
+| #define __EXTENSIONS__ 1
+| #define _ALL_SOURCE 1
+| #define _GNU_SOURCE 1
+| #define _POSIX_PTHREAD_SEMANTICS 1
+| #define _TANDEM_SOURCE 1
+| #define HAVE_STDLIB_H 1
+| #define HAVE_UNISTD_H 1
+| #define HAVE_MEMORY_H 1
+| #define HAVE_TERMIOS_H 1
+| #define HAVE_MALLOC_H 1
+| #define HAVE_LOCALE_H 1
+| #define HAVE_LANGINFO_H 1
+| #define HAVE_FCNTL_H 1
+| #define HAVE_SYS_FCNTL_H 1
+| #define HAVE_SYS_TYPES_H 1
+| #define HAVE_SYS_WAIT_H 1
+| #define HAVE_SYS_UTSNAME_H 1
+| #define HAVE_SYS_TIMES_H 1
+| #define HAVE_SYS_TIME_H 1
+| #define HAVE_UTIME_H 1
+| /* end confdefs.h.  */
+| #include <stdio.h>
+| #ifdef HAVE_SYS_TYPES_H
+| # include <sys/types.h>
+| #endif
+| #ifdef HAVE_SYS_STAT_H
+| # include <sys/stat.h>
+| #endif
+| #ifdef STDC_HEADERS
+| # include <stdlib.h>
+| # include <stddef.h>
+| #else
+| # ifdef HAVE_STDLIB_H
+| #  include <stdlib.h>
+| # endif
+| #endif
+| #ifdef HAVE_STRING_H
+| # if !defined STDC_HEADERS && defined HAVE_MEMORY_H
+| #  include <memory.h>
+| # endif
+| # include <string.h>
+| #endif
+| #ifdef HAVE_STRINGS_H
+| # include <strings.h>
+| #endif
+| #ifdef HAVE_INTTYPES_H
+| # include <inttypes.h>
+| #endif
+| #ifdef HAVE_STDINT_H
+| # include <stdint.h>
+| #endif
+| #ifdef HAVE_UNISTD_H
+| # include <unistd.h>
+| #endif
+| #include <floatingpoint.h>
+configure:4943: result: no
+configure:4943: checking floatingpoint.h presence
+configure:4943: gcc -E  conftest.c
+conftest.c:39:27: fatal error: floatingpoint.h: No such file or directory
+compilation terminated.
+configure:4943: $? = 1
+configure: failed program was:
+| /* confdefs.h */
+| #define PACKAGE_NAME ""
+| #define PACKAGE_TARNAME ""
+| #define PACKAGE_VERSION ""
+| #define PACKAGE_STRING ""
+| #define PACKAGE_BUGREPORT ""
+| #define PACKAGE_URL ""
+| #define STDC_HEADERS 1
+| #define HAVE_SYS_TYPES_H 1
+| #define HAVE_SYS_STAT_H 1
+| #define HAVE_STDLIB_H 1
+| #define HAVE_STRING_H 1
+| #define HAVE_MEMORY_H 1
+| #define HAVE_STRINGS_H 1
+| #define HAVE_INTTYPES_H 1
+| #define HAVE_STDINT_H 1
+| #define HAVE_UNISTD_H 1
+| #define __EXTENSIONS__ 1
+| #define _ALL_SOURCE 1
+| #define _GNU_SOURCE 1
+| #define _POSIX_PTHREAD_SEMANTICS 1
+| #define _TANDEM_SOURCE 1
+| #define HAVE_STDLIB_H 1
+| #define HAVE_UNISTD_H 1
+| #define HAVE_MEMORY_H 1
+| #define HAVE_TERMIOS_H 1
+| #define HAVE_MALLOC_H 1
+| #define HAVE_LOCALE_H 1
+| #define HAVE_LANGINFO_H 1
+| #define HAVE_FCNTL_H 1
+| #define HAVE_SYS_FCNTL_H 1
+| #define HAVE_SYS_TYPES_H 1
+| #define HAVE_SYS_WAIT_H 1
+| #define HAVE_SYS_UTSNAME_H 1
+| #define HAVE_SYS_TIMES_H 1
+| #define HAVE_SYS_TIME_H 1
+| #define HAVE_UTIME_H 1
+| /* end confdefs.h.  */
+| #include <floatingpoint.h>
+configure:4943: result: no
+configure:4943: checking for floatingpoint.h
+configure:4943: result: no
+configure:4943: checking ieeefp.h usability
+configure:4943: gcc -c -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2  conftest.c >&5
+conftest.c:72:20: fatal error: ieeefp.h: No such file or directory
+compilation terminated.
+configure:4943: $? = 1
+configure: failed program was:
+| /* confdefs.h */
+| #define PACKAGE_NAME ""
+| #define PACKAGE_TARNAME ""
+| #define PACKAGE_VERSION ""
+| #define PACKAGE_STRING ""
+| #define PACKAGE_BUGREPORT ""
+| #define PACKAGE_URL ""
+| #define STDC_HEADERS 1
+| #define HAVE_SYS_TYPES_H 1
+| #define HAVE_SYS_STAT_H 1
+| #define HAVE_STDLIB_H 1
+| #define HAVE_STRING_H 1
+| #define HAVE_MEMORY_H 1
+| #define HAVE_STRINGS_H 1
+| #define HAVE_INTTYPES_H 1
+| #define HAVE_STDINT_H 1
+| #define HAVE_UNISTD_H 1
+| #define __EXTENSIONS__ 1
+| #define _ALL_SOURCE 1
+| #define _GNU_SOURCE 1
+| #define _POSIX_PTHREAD_SEMANTICS 1
+| #define _TANDEM_SOURCE 1
+| #define HAVE_STDLIB_H 1
+| #define HAVE_UNISTD_H 1
+| #define HAVE_MEMORY_H 1
+| #define HAVE_TERMIOS_H 1
+| #define HAVE_MALLOC_H 1
+| #define HAVE_LOCALE_H 1
+| #define HAVE_LANGINFO_H 1
+| #define HAVE_FCNTL_H 1
+| #define HAVE_SYS_FCNTL_H 1
+| #define HAVE_SYS_TYPES_H 1
+| #define HAVE_SYS_WAIT_H 1
+| #define HAVE_SYS_UTSNAME_H 1
+| #define HAVE_SYS_TIMES_H 1
+| #define HAVE_SYS_TIME_H 1
+| #define HAVE_UTIME_H 1
+| /* end confdefs.h.  */
+| #include <stdio.h>
+| #ifdef HAVE_SYS_TYPES_H
+| # include <sys/types.h>
+| #endif
+| #ifdef HAVE_SYS_STAT_H
+| # include <sys/stat.h>
+| #endif
+| #ifdef STDC_HEADERS
+| # include <stdlib.h>
+| # include <stddef.h>
+| #else
+| # ifdef HAVE_STDLIB_H
+| #  include <stdlib.h>
+| # endif
+| #endif
+| #ifdef HAVE_STRING_H
+| # if !defined STDC_HEADERS && defined HAVE_MEMORY_H
+| #  include <memory.h>
+| # endif
+| # include <string.h>
+| #endif
+| #ifdef HAVE_STRINGS_H
+| # include <strings.h>
+| #endif
+| #ifdef HAVE_INTTYPES_H
+| # include <inttypes.h>
+| #endif
+| #ifdef HAVE_STDINT_H
+| # include <stdint.h>
+| #endif
+| #ifdef HAVE_UNISTD_H
+| # include <unistd.h>
+| #endif
+| #include <ieeefp.h>
+configure:4943: result: no
+configure:4943: checking ieeefp.h presence
+configure:4943: gcc -E  conftest.c
+conftest.c:39:20: fatal error: ieeefp.h: No such file or directory
+compilation terminated.
+configure:4943: $? = 1
+configure: failed program was:
+| /* confdefs.h */
+| #define PACKAGE_NAME ""
+| #define PACKAGE_TARNAME ""
+| #define PACKAGE_VERSION ""
+| #define PACKAGE_STRING ""
+| #define PACKAGE_BUGREPORT ""
+| #define PACKAGE_URL ""
+| #define STDC_HEADERS 1
+| #define HAVE_SYS_TYPES_H 1
+| #define HAVE_SYS_STAT_H 1
+| #define HAVE_STDLIB_H 1
+| #define HAVE_STRING_H 1
+| #define HAVE_MEMORY_H 1
+| #define HAVE_STRINGS_H 1
+| #define HAVE_INTTYPES_H 1
+| #define HAVE_STDINT_H 1
+| #define HAVE_UNISTD_H 1
+| #define __EXTENSIONS__ 1
+| #define _ALL_SOURCE 1
+| #define _GNU_SOURCE 1
+| #define _POSIX_PTHREAD_SEMANTICS 1
+| #define _TANDEM_SOURCE 1
+| #define HAVE_STDLIB_H 1
+| #define HAVE_UNISTD_H 1
+| #define HAVE_MEMORY_H 1
+| #define HAVE_TERMIOS_H 1
+| #define HAVE_MALLOC_H 1
+| #define HAVE_LOCALE_H 1
+| #define HAVE_LANGINFO_H 1
+| #define HAVE_FCNTL_H 1
+| #define HAVE_SYS_FCNTL_H 1
+| #define HAVE_SYS_TYPES_H 1
+| #define HAVE_SYS_WAIT_H 1
+| #define HAVE_SYS_UTSNAME_H 1
+| #define HAVE_SYS_TIMES_H 1
+| #define HAVE_SYS_TIME_H 1
+| #define HAVE_UTIME_H 1
+| /* end confdefs.h.  */
+| #include <ieeefp.h>
+configure:4943: result: no
+configure:4943: checking for ieeefp.h
+configure:4943: result: no
+configure:4943: checking nan.h usability
+configure:4943: gcc -c -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2  conftest.c >&5
+conftest.c:72:17: fatal error: nan.h: No such file or directory
+compilation terminated.
+configure:4943: $? = 1
+configure: failed program was:
+| /* confdefs.h */
+| #define PACKAGE_NAME ""
+| #define PACKAGE_TARNAME ""
+| #define PACKAGE_VERSION ""
+| #define PACKAGE_STRING ""
+| #define PACKAGE_BUGREPORT ""
+| #define PACKAGE_URL ""
+| #define STDC_HEADERS 1
+| #define HAVE_SYS_TYPES_H 1
+| #define HAVE_SYS_STAT_H 1
+| #define HAVE_STDLIB_H 1
+| #define HAVE_STRING_H 1
+| #define HAVE_MEMORY_H 1
+| #define HAVE_STRINGS_H 1
+| #define HAVE_INTTYPES_H 1
+| #define HAVE_STDINT_H 1
+| #define HAVE_UNISTD_H 1
+| #define __EXTENSIONS__ 1
+| #define _ALL_SOURCE 1
+| #define _GNU_SOURCE 1
+| #define _POSIX_PTHREAD_SEMANTICS 1
+| #define _TANDEM_SOURCE 1
+| #define HAVE_STDLIB_H 1
+| #define HAVE_UNISTD_H 1
+| #define HAVE_MEMORY_H 1
+| #define HAVE_TERMIOS_H 1
+| #define HAVE_MALLOC_H 1
+| #define HAVE_LOCALE_H 1
+| #define HAVE_LANGINFO_H 1
+| #define HAVE_FCNTL_H 1
+| #define HAVE_SYS_FCNTL_H 1
+| #define HAVE_SYS_TYPES_H 1
+| #define HAVE_SYS_WAIT_H 1
+| #define HAVE_SYS_UTSNAME_H 1
+| #define HAVE_SYS_TIMES_H 1
+| #define HAVE_SYS_TIME_H 1
+| #define HAVE_UTIME_H 1
+| /* end confdefs.h.  */
+| #include <stdio.h>
+| #ifdef HAVE_SYS_TYPES_H
+| # include <sys/types.h>
+| #endif
+| #ifdef HAVE_SYS_STAT_H
+| # include <sys/stat.h>
+| #endif
+| #ifdef STDC_HEADERS
+| # include <stdlib.h>
+| # include <stddef.h>
+| #else
+| # ifdef HAVE_STDLIB_H
+| #  include <stdlib.h>
+| # endif
+| #endif
+| #ifdef HAVE_STRING_H
+| # if !defined STDC_HEADERS && defined HAVE_MEMORY_H
+| #  include <memory.h>
+| # endif
+| # include <string.h>
+| #endif
+| #ifdef HAVE_STRINGS_H
+| # include <strings.h>
+| #endif
+| #ifdef HAVE_INTTYPES_H
+| # include <inttypes.h>
+| #endif
+| #ifdef HAVE_STDINT_H
+| # include <stdint.h>
+| #endif
+| #ifdef HAVE_UNISTD_H
+| # include <unistd.h>
+| #endif
+| #include <nan.h>
+configure:4943: result: no
+configure:4943: checking nan.h presence
+configure:4943: gcc -E  conftest.c
+conftest.c:39:17: fatal error: nan.h: No such file or directory
+compilation terminated.
+configure:4943: $? = 1
+configure: failed program was:
+| /* confdefs.h */
+| #define PACKAGE_NAME ""
+| #define PACKAGE_TARNAME ""
+| #define PACKAGE_VERSION ""
+| #define PACKAGE_STRING ""
+| #define PACKAGE_BUGREPORT ""
+| #define PACKAGE_URL ""
+| #define STDC_HEADERS 1
+| #define HAVE_SYS_TYPES_H 1
+| #define HAVE_SYS_STAT_H 1
+| #define HAVE_STDLIB_H 1
+| #define HAVE_STRING_H 1
+| #define HAVE_MEMORY_H 1
+| #define HAVE_STRINGS_H 1
+| #define HAVE_INTTYPES_H 1
+| #define HAVE_STDINT_H 1
+| #define HAVE_UNISTD_H 1
+| #define __EXTENSIONS__ 1
+| #define _ALL_SOURCE 1
+| #define _GNU_SOURCE 1
+| #define _POSIX_PTHREAD_SEMANTICS 1
+| #define _TANDEM_SOURCE 1
+| #define HAVE_STDLIB_H 1
+| #define HAVE_UNISTD_H 1
+| #define HAVE_MEMORY_H 1
+| #define HAVE_TERMIOS_H 1
+| #define HAVE_MALLOC_H 1
+| #define HAVE_LOCALE_H 1
+| #define HAVE_LANGINFO_H 1
+| #define HAVE_FCNTL_H 1
+| #define HAVE_SYS_FCNTL_H 1
+| #define HAVE_SYS_TYPES_H 1
+| #define HAVE_SYS_WAIT_H 1
+| #define HAVE_SYS_UTSNAME_H 1
+| #define HAVE_SYS_TIMES_H 1
+| #define HAVE_SYS_TIME_H 1
+| #define HAVE_UTIME_H 1
+| /* end confdefs.h.  */
+| #include <nan.h>
+configure:4943: result: no
+configure:4943: checking for nan.h
+configure:4943: result: no
+configure:4943: checking fenv.h usability
+configure:4943: gcc -c -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2  conftest.c >&5
+configure:4943: $? = 0
+configure:4943: result: yes
+configure:4943: checking fenv.h presence
+configure:4943: gcc -E  conftest.c
+configure:4943: $? = 0
+configure:4943: result: yes
+configure:4943: checking for fenv.h
+configure:4943: result: yes
+configure:4943: checking sys/mman.h usability
+configure:4943: gcc -c -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2  conftest.c >&5
+configure:4943: $? = 0
+configure:4943: result: yes
+configure:4943: checking sys/mman.h presence
+configure:4943: gcc -E  conftest.c
+configure:4943: $? = 0
+configure:4943: result: yes
+configure:4943: checking for sys/mman.h
+configure:4943: result: yes
+configure:4943: checking sys/socket.h usability
+configure:4943: gcc -c -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2  conftest.c >&5
+configure:4943: $? = 0
+configure:4943: result: yes
+configure:4943: checking sys/socket.h presence
+configure:4943: gcc -E  conftest.c
+configure:4943: $? = 0
+configure:4943: result: yes
+configure:4943: checking for sys/socket.h
+configure:4943: result: yes
+configure:4943: checking netinet/in.h usability
+configure:4943: gcc -c -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2  conftest.c >&5
+configure:4943: $? = 0
+configure:4943: result: yes
+configure:4943: checking netinet/in.h presence
+configure:4943: gcc -E  conftest.c
+configure:4943: $? = 0
+configure:4943: result: yes
+configure:4943: checking for netinet/in.h
+configure:4943: result: yes
+configure:4943: checking arpa/inet.h usability
+configure:4943: gcc -c -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2  conftest.c >&5
+configure:4943: $? = 0
+configure:4943: result: yes
+configure:4943: checking arpa/inet.h presence
+configure:4943: gcc -E  conftest.c
+configure:4943: $? = 0
+configure:4943: result: yes
+configure:4943: checking for arpa/inet.h
+configure:4943: result: yes
+configure:4943: checking sys/un.h usability
+configure:4943: gcc -c -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2  conftest.c >&5
+configure:4943: $? = 0
+configure:4943: result: yes
+configure:4943: checking sys/un.h presence
+configure:4943: gcc -E  conftest.c
+configure:4943: $? = 0
+configure:4943: result: yes
+configure:4943: checking for sys/un.h
+configure:4943: result: yes
+configure:4943: checking sys/resource.h usability
+configure:4943: gcc -c -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2  conftest.c >&5
+configure:4943: $? = 0
+configure:4943: result: yes
+configure:4943: checking sys/resource.h presence
+configure:4943: gcc -E  conftest.c
+configure:4943: $? = 0
+configure:4943: result: yes
+configure:4943: checking for sys/resource.h
+configure:4943: result: yes
+configure:4954: checking for mode_t
+configure:4954: gcc -c -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2  conftest.c >&5
+configure:4954: $? = 0
+configure:4954: gcc -c -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2  conftest.c >&5
+conftest.c: In function 'main':
+conftest.c:82:21: error: expected expression before ')' token
+configure:4954: $? = 1
+configure: failed program was:
+| /* confdefs.h */
+| #define PACKAGE_NAME ""
+| #define PACKAGE_TARNAME ""
+| #define PACKAGE_VERSION ""
+| #define PACKAGE_STRING ""
+| #define PACKAGE_BUGREPORT ""
+| #define PACKAGE_URL ""
+| #define STDC_HEADERS 1
+| #define HAVE_SYS_TYPES_H 1
+| #define HAVE_SYS_STAT_H 1
+| #define HAVE_STDLIB_H 1
+| #define HAVE_STRING_H 1
+| #define HAVE_MEMORY_H 1
+| #define HAVE_STRINGS_H 1
+| #define HAVE_INTTYPES_H 1
+| #define HAVE_STDINT_H 1
+| #define HAVE_UNISTD_H 1
+| #define __EXTENSIONS__ 1
+| #define _ALL_SOURCE 1
+| #define _GNU_SOURCE 1
+| #define _POSIX_PTHREAD_SEMANTICS 1
+| #define _TANDEM_SOURCE 1
+| #define HAVE_STDLIB_H 1
+| #define HAVE_UNISTD_H 1
+| #define HAVE_MEMORY_H 1
+| #define HAVE_TERMIOS_H 1
+| #define HAVE_MALLOC_H 1
+| #define HAVE_LOCALE_H 1
+| #define HAVE_LANGINFO_H 1
+| #define HAVE_FCNTL_H 1
+| #define HAVE_SYS_FCNTL_H 1
+| #define HAVE_SYS_TYPES_H 1
+| #define HAVE_SYS_WAIT_H 1
+| #define HAVE_SYS_UTSNAME_H 1
+| #define HAVE_SYS_TIMES_H 1
+| #define HAVE_SYS_TIME_H 1
+| #define HAVE_UTIME_H 1
+| #define HAVE_FENV_H 1
+| #define HAVE_SYS_MMAN_H 1
+| #define HAVE_SYS_SOCKET_H 1
+| #define HAVE_NETINET_IN_H 1
+| #define HAVE_ARPA_INET_H 1
+| #define HAVE_SYS_UN_H 1
+| #define HAVE_SYS_RESOURCE_H 1
+| /* end confdefs.h.  */
+| #include <stdio.h>
+| #ifdef HAVE_SYS_TYPES_H
+| # include <sys/types.h>
+| #endif
+| #ifdef HAVE_SYS_STAT_H
+| # include <sys/stat.h>
+| #endif
+| #ifdef STDC_HEADERS
+| # include <stdlib.h>
+| # include <stddef.h>
+| #else
+| # ifdef HAVE_STDLIB_H
+| #  include <stdlib.h>
+| # endif
+| #endif
+| #ifdef HAVE_STRING_H
+| # if !defined STDC_HEADERS && defined HAVE_MEMORY_H
+| #  include <memory.h>
+| # endif
+| # include <string.h>
+| #endif
+| #ifdef HAVE_STRINGS_H
+| # include <strings.h>
+| #endif
+| #ifdef HAVE_INTTYPES_H
+| # include <inttypes.h>
+| #endif
+| #ifdef HAVE_STDINT_H
+| # include <stdint.h>
+| #endif
+| #ifdef HAVE_UNISTD_H
+| # include <unistd.h>
+| #endif
+| int
+| main ()
+| {
+| if (sizeof ((mode_t)))
+| 	    return 0;
+|   ;
+|   return 0;
+| }
+configure:4954: result: yes
+configure:4965: checking for pid_t
+configure:4965: gcc -c -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2  conftest.c >&5
+configure:4965: $? = 0
+configure:4965: gcc -c -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2  conftest.c >&5
+conftest.c: In function 'main':
+conftest.c:82:20: error: expected expression before ')' token
+configure:4965: $? = 1
+configure: failed program was:
+| /* confdefs.h */
+| #define PACKAGE_NAME ""
+| #define PACKAGE_TARNAME ""
+| #define PACKAGE_VERSION ""
+| #define PACKAGE_STRING ""
+| #define PACKAGE_BUGREPORT ""
+| #define PACKAGE_URL ""
+| #define STDC_HEADERS 1
+| #define HAVE_SYS_TYPES_H 1
+| #define HAVE_SYS_STAT_H 1
+| #define HAVE_STDLIB_H 1
+| #define HAVE_STRING_H 1
+| #define HAVE_MEMORY_H 1
+| #define HAVE_STRINGS_H 1
+| #define HAVE_INTTYPES_H 1
+| #define HAVE_STDINT_H 1
+| #define HAVE_UNISTD_H 1
+| #define __EXTENSIONS__ 1
+| #define _ALL_SOURCE 1
+| #define _GNU_SOURCE 1
+| #define _POSIX_PTHREAD_SEMANTICS 1
+| #define _TANDEM_SOURCE 1
+| #define HAVE_STDLIB_H 1
+| #define HAVE_UNISTD_H 1
+| #define HAVE_MEMORY_H 1
+| #define HAVE_TERMIOS_H 1
+| #define HAVE_MALLOC_H 1
+| #define HAVE_LOCALE_H 1
+| #define HAVE_LANGINFO_H 1
+| #define HAVE_FCNTL_H 1
+| #define HAVE_SYS_FCNTL_H 1
+| #define HAVE_SYS_TYPES_H 1
+| #define HAVE_SYS_WAIT_H 1
+| #define HAVE_SYS_UTSNAME_H 1
+| #define HAVE_SYS_TIMES_H 1
+| #define HAVE_SYS_TIME_H 1
+| #define HAVE_UTIME_H 1
+| #define HAVE_FENV_H 1
+| #define HAVE_SYS_MMAN_H 1
+| #define HAVE_SYS_SOCKET_H 1
+| #define HAVE_NETINET_IN_H 1
+| #define HAVE_ARPA_INET_H 1
+| #define HAVE_SYS_UN_H 1
+| #define HAVE_SYS_RESOURCE_H 1
+| /* end confdefs.h.  */
+| #include <stdio.h>
+| #ifdef HAVE_SYS_TYPES_H
+| # include <sys/types.h>
+| #endif
+| #ifdef HAVE_SYS_STAT_H
+| # include <sys/stat.h>
+| #endif
+| #ifdef STDC_HEADERS
+| # include <stdlib.h>
+| # include <stddef.h>
+| #else
+| # ifdef HAVE_STDLIB_H
+| #  include <stdlib.h>
+| # endif
+| #endif
+| #ifdef HAVE_STRING_H
+| # if !defined STDC_HEADERS && defined HAVE_MEMORY_H
+| #  include <memory.h>
+| # endif
+| # include <string.h>
+| #endif
+| #ifdef HAVE_STRINGS_H
+| # include <strings.h>
+| #endif
+| #ifdef HAVE_INTTYPES_H
+| # include <inttypes.h>
+| #endif
+| #ifdef HAVE_STDINT_H
+| # include <stdint.h>
+| #endif
+| #ifdef HAVE_UNISTD_H
+| # include <unistd.h>
+| #endif
+| int
+| main ()
+| {
+| if (sizeof ((pid_t)))
+| 	    return 0;
+|   ;
+|   return 0;
+| }
+configure:4965: result: yes
+configure:4976: checking for uid_t in sys/types.h
+configure:4995: result: yes
+configure:5007: checking for socklen_t
+configure:5016: result: yes
+configure:5031: checking for dirent.h that defines DIR
+configure:5050: gcc -c -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2  conftest.c >&5
+configure:5050: $? = 0
+configure:5058: result: yes
+configure:5071: checking for library containing opendir
+configure:5102: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c  >&5
+configure:5102: $? = 0
+configure:5119: result: none required
+configure:5230: checking for memset
+configure:5230: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c  >&5
+conftest.c:70:6: warning: conflicting types for built-in function 'memset'
+configure:5230: $? = 0
+configure:5230: result: yes
+configure:5230: checking for memcpy
+configure:5230: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c  >&5
+conftest.c:71:6: warning: conflicting types for built-in function 'memcpy'
+configure:5230: $? = 0
+configure:5230: result: yes
+configure:5230: checking for putenv
+configure:5230: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c  >&5
+configure:5230: $? = 0
+configure:5230: result: yes
+configure:5230: checking for getcwd
+configure:5230: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c  >&5
+configure:5230: $? = 0
+configure:5230: result: yes
+configure:5230: checking for setlocale
+configure:5230: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c  >&5
+configure:5230: $? = 0
+configure:5230: result: yes
+configure:5230: checking for tcgetattr
+configure:5230: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c  >&5
+configure:5230: $? = 0
+configure:5230: result: yes
+configure:5230: checking for tcsetattr
+configure:5230: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c  >&5
+configure:5230: $? = 0
+configure:5230: result: yes
+configure:5230: checking for cfgetospeed
+configure:5230: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c  >&5
+configure:5230: $? = 0
+configure:5230: result: yes
+configure:5230: checking for sigaction
+configure:5230: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c  >&5
+configure:5230: $? = 0
+configure:5230: result: yes
+configure:5230: checking for sigemptyset
+configure:5230: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c  >&5
+configure:5230: $? = 0
+configure:5230: result: yes
+configure:5230: checking for sigprocmask
+configure:5230: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c  >&5
+configure:5230: $? = 0
+configure:5230: result: yes
+configure:5230: checking for sigaddset
+configure:5230: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c  >&5
+configure:5230: $? = 0
+configure:5230: result: yes
+configure:5230: checking for alarm
+configure:5230: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c  >&5
+configure:5230: $? = 0
+configure:5230: result: yes
+configure:5230: checking for pause
+configure:5230: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c  >&5
+configure:5230: $? = 0
+configure:5230: result: yes
+configure:5230: checking for vfscanf
+configure:5230: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c  >&5
+conftest.c:84:6: warning: conflicting types for built-in function 'vfscanf'
+configure:5230: $? = 0
+configure:5230: result: yes
+configure:5230: checking for lstat
+configure:5230: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c  >&5
+configure:5230: $? = 0
+configure:5230: result: yes
+configure:5230: checking for readlink
+configure:5230: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c  >&5
+configure:5230: $? = 0
+configure:5230: result: yes
+configure:5230: checking for symlink
+configure:5230: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c  >&5
+configure:5230: $? = 0
+configure:5230: result: yes
+configure:5230: checking for link
+configure:5230: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c  >&5
+configure:5230: $? = 0
+configure:5230: result: yes
+configure:5230: checking for kill
+configure:5230: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c  >&5
+configure:5230: $? = 0
+configure:5230: result: yes
+configure:5230: checking for killpg
+configure:5230: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c  >&5
+configure:5230: $? = 0
+configure:5230: result: yes
+configure:5230: checking for snprintf
+configure:5230: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c  >&5
+conftest.c:91:6: warning: conflicting types for built-in function 'snprintf'
+configure:5230: $? = 0
+configure:5230: result: yes
+configure:5230: checking for vsnprintf
+configure:5230: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c  >&5
+conftest.c:92:6: warning: conflicting types for built-in function 'vsnprintf'
+configure:5230: $? = 0
+configure:5230: result: yes
+configure:5230: checking for getppid
+configure:5230: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c  >&5
+configure:5230: $? = 0
+configure:5230: result: yes
+configure:5230: checking for getegid
+configure:5230: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c  >&5
+configure:5230: $? = 0
+configure:5230: result: yes
+configure:5230: checking for geteuid
+configure:5230: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c  >&5
+configure:5230: $? = 0
+configure:5230: result: yes
+configure:5230: checking for getuid
+configure:5230: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c  >&5
+configure:5230: $? = 0
+configure:5230: result: yes
+configure:5230: checking for getgid
+configure:5230: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c  >&5
+configure:5230: $? = 0
+configure:5230: result: yes
+configure:5230: checking for setgid
+configure:5230: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c  >&5
+configure:5230: $? = 0
+configure:5230: result: yes
+configure:5230: checking for setuid
+configure:5230: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c  >&5
+configure:5230: $? = 0
+configure:5230: result: yes
+configure:5230: checking for setpgid
+configure:5230: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c  >&5
+configure:5230: $? = 0
+configure:5230: result: yes
+configure:5230: checking for getpgid
+configure:5230: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c  >&5
+configure:5230: $? = 0
+configure:5230: result: yes
+configure:5230: checking for setpgrp
+configure:5230: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c  >&5
+configure:5230: $? = 0
+configure:5230: result: yes
+configure:5230: checking for getpgrp
+configure:5230: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c  >&5
+configure:5230: $? = 0
+configure:5230: result: yes
+configure:5230: checking for setsid
+configure:5230: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c  >&5
+configure:5230: $? = 0
+configure:5230: result: yes
+configure:5230: checking for getsid
+configure:5230: gcc -o conftest -g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2   conftest.c  >&5
+configure:5230: $? = 0
+configure:5230: result: yes
+configure:5230: checking for mmap
+
+## ---------------- ##
+## Cache variables. ##
+## ---------------- ##
+
+ac_cv_build=i486-pc-linux-gnu
+ac_cv_c_compiler_gnu=yes
+ac_cv_env_CC_set=
+ac_cv_env_CC_value=
+ac_cv_env_CFLAGS_set=set
+ac_cv_env_CFLAGS_value='-g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2'
+ac_cv_env_CPPFLAGS_set=set
+ac_cv_env_CPPFLAGS_value=
+ac_cv_env_CPP_set=
+ac_cv_env_CPP_value=
+ac_cv_env_LDFLAGS_set=set
+ac_cv_env_LDFLAGS_value=
+ac_cv_env_LIBS_set=
+ac_cv_env_LIBS_value=
+ac_cv_env_XMKMF_set=
+ac_cv_env_XMKMF_value=
+ac_cv_env_build_alias_set=set
+ac_cv_env_build_alias_value=i486-linux-gnu
+ac_cv_env_host_alias_set=
+ac_cv_env_host_alias_value=
+ac_cv_env_target_alias_set=
+ac_cv_env_target_alias_value=
+ac_cv_func_alarm=yes
+ac_cv_func_cfgetospeed=yes
+ac_cv_func_connect=yes
+ac_cv_func_getcwd=yes
+ac_cv_func_getegid=yes
+ac_cv_func_geteuid=yes
+ac_cv_func_getgid=yes
+ac_cv_func_gethostbyname=yes
+ac_cv_func_getpgid=yes
+ac_cv_func_getpgrp=yes
+ac_cv_func_getppid=yes
+ac_cv_func_getsid=yes
+ac_cv_func_getuid=yes
+ac_cv_func_kill=yes
+ac_cv_func_killpg=yes
+ac_cv_func_link=yes
+ac_cv_func_lstat=yes
+ac_cv_func_memcpy=yes
+ac_cv_func_memset=yes
+ac_cv_func_pause=yes
+ac_cv_func_putenv=yes
+ac_cv_func_readlink=yes
+ac_cv_func_remove=yes
+ac_cv_func_setgid=yes
+ac_cv_func_setlocale=yes
+ac_cv_func_setpgid=yes
+ac_cv_func_setpgrp=yes
+ac_cv_func_setsid=yes
+ac_cv_func_setuid=yes
+ac_cv_func_shmat=yes
+ac_cv_func_sigaction=yes
+ac_cv_func_sigaddset=yes
+ac_cv_func_sigemptyset=yes
+ac_cv_func_sigprocmask=yes
+ac_cv_func_snprintf=yes
+ac_cv_func_symlink=yes
+ac_cv_func_tcgetattr=yes
+ac_cv_func_tcsetattr=yes
+ac_cv_func_vfscanf=yes
+ac_cv_func_vsnprintf=yes
+ac_cv_have_x='have_x=yes	ac_x_includes='\'''\''	ac_x_libraries='\'''\'''
+ac_cv_header_arpa_inet_h=yes
+ac_cv_header_dirent_dirent_h=yes
+ac_cv_header_fcntl_h=yes
+ac_cv_header_fenv_h=yes
+ac_cv_header_floatingpoint_h=no
+ac_cv_header_ieeefp_h=no
+ac_cv_header_inttypes_h=yes
+ac_cv_header_langinfo_h=yes
+ac_cv_header_locale_h=yes
+ac_cv_header_malloc_h=yes
+ac_cv_header_memory_h=yes
+ac_cv_header_minix_config_h=no
+ac_cv_header_nan_h=no
+ac_cv_header_netinet_in_h=yes
+ac_cv_header_stdc=yes
+ac_cv_header_stdint_h=yes
+ac_cv_header_stdlib_h=yes
+ac_cv_header_string_h=yes
+ac_cv_header_strings_h=yes
+ac_cv_header_sys_fcntl_h=yes
+ac_cv_header_sys_mman_h=yes
+ac_cv_header_sys_resource_h=yes
+ac_cv_header_sys_socket_h=yes
+ac_cv_header_sys_stat_h=yes
+ac_cv_header_sys_time_h=yes
+ac_cv_header_sys_times_h=yes
+ac_cv_header_sys_types_h=yes
+ac_cv_header_sys_un_h=yes
+ac_cv_header_sys_utsname_h=yes
+ac_cv_header_sys_wait_h=yes
+ac_cv_header_termios_h=yes
+ac_cv_header_unistd_h=yes
+ac_cv_header_utime_h=yes
+ac_cv_host=i486-pc-linux-gnu
+ac_cv_lib_ICE_IceConnectionNumber=yes
+ac_cv_objext=o
+ac_cv_path_EGREP='/bin/grep -E'
+ac_cv_path_GREP=/bin/grep
+ac_cv_path_install='/usr/bin/install -c'
+ac_cv_prog_CPP='gcc -E'
+ac_cv_prog_ac_ct_CC=gcc
+ac_cv_prog_ac_ct_RANLIB=ranlib
+ac_cv_prog_cc_c89=
+ac_cv_prog_cc_g=yes
+ac_cv_prog_gcc_traditional=no
+ac_cv_prog_make_make_set=yes
+ac_cv_safe_to_define___extensions__=yes
+ac_cv_search_opendir='none required'
+ac_cv_search_strerror='none required'
+ac_cv_type_mode_t=yes
+ac_cv_type_pid_t=yes
+ac_cv_type_uid_t=yes
+
+## ----------------- ##
+## Output variables. ##
+## ----------------- ##
+
+CC='gcc'
+CC_SHARED=''
+CFLAGS='-g -fno-strength-reduce -D_REENTRANT -D_XOPEN_SOURCE=500  -O2'
+CONFIG_DIR='/home/alastair/slang/slang2-2.2.4'
+CPP='gcc -E'
+CPPFLAGS=''
+DEFS=''
+DLL_IMPLIB_NAME=''
+DYNAMIC_LINK_FLAGS=''
+DYNAMIC_LINK_LIB=''
+ECHO_C=''
+ECHO_N='-n'
+ECHO_T=''
+EGREP='/bin/grep -E'
+ELFDIR=''
+ELFLIB=''
+ELFLIB_BUILD_NAME=''
+ELFLIB_MAJOR=''
+ELFLIB_MAJOR_MINOR=''
+ELFLIB_MAJOR_MINOR_MICRO=''
+ELF_CC=''
+ELF_CFLAGS=''
+ELF_DEP_LIBS=''
+ELF_LINK=''
+ELF_LINK_CMD=''
+EXEEXT=''
+GNU_READLINE='0'
+GREP='/bin/grep'
+ICONV_INC=''
+ICONV_INC_DIR=''
+ICONV_LIB=''
+ICONV_LIB_DIR=''
+ICONV_MODULE=''
+INSTALL_DATA='${INSTALL} -m 644'
+INSTALL_ELFLIB_TARGET=''
+INSTALL_MODULE=''
+INSTALL_PROGRAM='${INSTALL}'
+INSTALL_SCRIPT='${INSTALL}'
+LDFLAGS=''
+LIBOBJS=''
+LIBS=''
+LIB_READLINE='# -lreadline'
+LTLIBOBJS=''
+MISC_TERMINFO_DIRS=''
+M_LIB=''
+OBJDIR=''
+OBJEXT='o'
+ONIG_INC=''
+ONIG_INC_DIR=''
+ONIG_LIB=''
+ONIG_LIB_DIR=''
+ONIG_MODULE=''
+PACKAGE_BUGREPORT=''
+PACKAGE_NAME=''
+PACKAGE_STRING=''
+PACKAGE_TARNAME=''
+PACKAGE_URL=''
+PACKAGE_VERSION=''
+PATH_SEPARATOR=':'
+PCRE_INC=''
+PCRE_INC_DIR=''
+PCRE_LIB=''
+PCRE_LIB_DIR=''
+PCRE_MODULE=''
+PNG_INC=''
+PNG_INC_DIR=''
+PNG_LIB=''
+PNG_LIB_DIR=''
+PNG_MODULE=''
+PROGRAM_CFILES=''
+PROGRAM_ELFOBJECTS=''
+PROGRAM_HFILES=''
+PROGRAM_OBJECTS=''
+PROGRAM_OFILES=''
+RANLIB='ranlib'
+RPATH=''
+SET_MAKE=''
+SHELL='/bin/bash'
+SLANG_DLL_CFLAGS=''
+SLANG_LIB_FOR_MODULES=''
+SRCDIR=''
+SYSCONF_MODULE=''
+TERMCAP=''
+TERMINFO_LAYOUT='default'
+XMKMF=''
+X_CFLAGS=''
+X_EXTRA_LIBS=''
+X_LIBS=''
+X_PRE_LIBS=' -lSM -lICE'
+ZLIB_MODULE=''
+Z_INC=''
+Z_INC_DIR=''
+Z_LIB=''
+Z_LIB_DIR=''
+ac_ct_CC='gcc'
+bindir='${exec_prefix}/bin'
+build='i486-pc-linux-gnu'
+build_alias='i486-linux-gnu'
+build_cpu='i486'
+build_os='linux-gnu'
+build_vendor='pc'
+datadir='${datarootdir}'
+datarootdir='${prefix}/share'
+docdir='${datarootdir}/doc/${PACKAGE}'
+dvidir='${docdir}'
+exec_prefix='NONE'
+host='i486-pc-linux-gnu'
+host_alias=''
+host_cpu='i486'
+host_os='linux-gnu'
+host_vendor='pc'
+htmldir='${docdir}'
+includedir='${prefix}/include'
+infodir='${prefix}/share/info'
+libdir='${exec_prefix}/lib'
+libexecdir='${prefix}/lib/slang2'
+localedir='${datarootdir}/locale'
+localstatedir='/var'
+mandir='${prefix}/share/man'
+nc5config=''
+oldincludedir='/usr/include'
+pdfdir='${docdir}'
+pkgconfigdir='${libdir}/pkgconfig'
+prefix='/usr'
+program_transform_name='s,x,x,'
+psdir='${docdir}'
+sbindir='${exec_prefix}/sbin'
+sharedstatedir='${prefix}/com'
+slang_major_version=''
+slang_micro_version=''
+slang_minor_version=''
+slang_version=''
+sysconfdir='/etc'
+target_alias=''
+
+## ----------- ##
+## confdefs.h. ##
+## ----------- ##
+
+/* confdefs.h */
+#define PACKAGE_NAME ""
+#define PACKAGE_TARNAME ""
+#define PACKAGE_VERSION ""
+#define PACKAGE_STRING ""
+#define PACKAGE_BUGREPORT ""
+#define PACKAGE_URL ""
+#define STDC_HEADERS 1
+#define HAVE_SYS_TYPES_H 1
+#define HAVE_SYS_STAT_H 1
+#define HAVE_STDLIB_H 1
+#define HAVE_STRING_H 1
+#define HAVE_MEMORY_H 1
+#define HAVE_STRINGS_H 1
+#define HAVE_INTTYPES_H 1
+#define HAVE_STDINT_H 1
+#define HAVE_UNISTD_H 1
+#define __EXTENSIONS__ 1
+#define _ALL_SOURCE 1
+#define _GNU_SOURCE 1
+#define _POSIX_PTHREAD_SEMANTICS 1
+#define _TANDEM_SOURCE 1
+#define HAVE_STDLIB_H 1
+#define HAVE_UNISTD_H 1
+#define HAVE_MEMORY_H 1
+#define HAVE_TERMIOS_H 1
+#define HAVE_MALLOC_H 1
+#define HAVE_LOCALE_H 1
+#define HAVE_LANGINFO_H 1
+#define HAVE_FCNTL_H 1
+#define HAVE_SYS_FCNTL_H 1
+#define HAVE_SYS_TYPES_H 1
+#define HAVE_SYS_WAIT_H 1
+#define HAVE_SYS_UTSNAME_H 1
+#define HAVE_SYS_TIMES_H 1
+#define HAVE_SYS_TIME_H 1
+#define HAVE_UTIME_H 1
+#define HAVE_FENV_H 1
+#define HAVE_SYS_MMAN_H 1
+#define HAVE_SYS_SOCKET_H 1
+#define HAVE_NETINET_IN_H 1
+#define HAVE_ARPA_INET_H 1
+#define HAVE_SYS_UN_H 1
+#define HAVE_SYS_RESOURCE_H 1
+#define HAVE_DIRENT_H 1
+#define HAVE_MEMSET 1
+#define HAVE_MEMCPY 1
+#define HAVE_PUTENV 1
+#define HAVE_GETCWD 1
+#define HAVE_SETLOCALE 1
+#define HAVE_TCGETATTR 1
+#define HAVE_TCSETATTR 1
+#define HAVE_CFGETOSPEED 1
+#define HAVE_SIGACTION 1
+#define HAVE_SIGEMPTYSET 1
+#define HAVE_SIGPROCMASK 1
+#define HAVE_SIGADDSET 1
+#define HAVE_ALARM 1
+#define HAVE_PAUSE 1
+#define HAVE_VFSCANF 1
+#define HAVE_LSTAT 1
+#define HAVE_READLINK 1
+#define HAVE_SYMLINK 1
+#define HAVE_LINK 1
+#define HAVE_KILL 1
+#define HAVE_KILLPG 1
+#define HAVE_SNPRINTF 1
+#define HAVE_VSNPRINTF 1
+#define HAVE_GETPPID 1
+#define HAVE_GETEGID 1
+#define HAVE_GETEUID 1
+#define HAVE_GETUID 1
+#define HAVE_GETGID 1
+#define HAVE_SETGID 1
+#define HAVE_SETUID 1
+#define HAVE_SETPGID 1
+#define HAVE_GETPGID 1
+#define HAVE_SETPGRP 1
+#define HAVE_GETPGRP 1
+#define HAVE_SETSID 1
+#define HAVE_GETSID 1
+
+configure: caught signal 2
+configure: exit 1
